staging_service/app.py

```diff
@@ -12,10 +15,66 @@
 from .globus import assert_globusid_exists, is_globusid
 from .metadata import some_metadata, dir_info, add_upa, similar
 from .utils import Path, run_command, AclManager
+from .import_specifications.file_parser import (
+    ErrorType,
+    Error,
+    FileTypeResolution,
+    parse_import_specifications,
+)
+from .import_specifications.individual_parsers import parse_csv, parse_tsv, parse_excel
+from .autodetect.Mappings import CSV, TSV, EXCEL
 
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
 routes = web.RouteTableDef()
 VERSION = "1.2.0"
 
+_APP_JSON = "application/json"
+
+_IMPSPEC_FILE_TO_PARSER = {
+    CSV: parse_csv,
+    TSV: parse_tsv,
+    EXCEL: parse_excel,
+}
+
+# Types for the lambda parameters are all strings
```

Comment:
Maybe we should just keep functions with routes in this file, and move out anything doesn't have a route?